nomic/project.py

```diff
@@ -963,11 +967,62 @@ def get_projection_id():
 
     def __repr__(self):
         m = self.meta
-        return f"Nomic project: <{m}>"
+        return f"AtlasProject: <{m}>"
 
     def __str__(self):
         return "\n".join([str(projection) for index in self.indices for projection in index.projections])
 
+
+    def get_data(self, ids: List[str]) -> List[Dict]:
```

Comment:
Why shouldn't get_data include user-generated tags as part of the return signature? Isn't that part of the dataset?

Reply:
made a card